File: 231. Power of Two.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfTwo = function(n) {
    if (n <= 0) return false;
    var temp = Math.log10(n) / Math.log10(2);
    if (Math.round(temp) == temp) return true;
    return false;
};

File: 231. Power of Two.py
class Solution(object):
    def isPowerOfTwo(self, n):
        """
        :type n: int
        :rtype: bool
        """
        if n <= 0:
            return False
        temp = math.log10(n) / float(math.log10(2))
        return True if int(temp) == temp else False
